CMakeLists.txt

```diff
@@ -336,6 +331,24 @@ endif()
 check_function_exists(chmod HAVE_CHMOD)
 check_function_exists(umask HAVE_UMASK)
 
+if(NOT WIN32 AND NOT DEFINED 3DS AND NOT DEFINED PSP2 AND NOT DEFINED WII)
```

Comment:
That's...not what I meant. I meant something more like `check_include_file(pthread.h USE_PTHREADS)` so the variable gets unset if the header can't be found.

Reply:
CMake caches variables set by check_include_file/check_function_exists/etc, and as a result if I set USE_PTHREADS earlier on in the file, CMake will skip the test assuming that the check has already been run.

Reply:
Oh right. Well maybe go back to the old version but do the phtread.h check where set(USE_PTHREADS ON) used to be?